src/tests/results_tests/test_utils.py

```diff
@@ -0,0 +1,66 @@
+from triage.component.results_schema.utils import (
+    save_experiment_and_get_hash,
+    associate_models_with_experiment,
+    associate_matrices_with_experiment,
+    missing_model_hashes,
+    missing_matrix_uuids,
+)
+
+from triage.component.results_schema.schema import Matrix, Model
+from triage.component.catwalk.db import ensure_db
+from sqlalchemy import create_engine
+import testing.postgresql
+
+
+def test_save_experiment_and_get_hash():
```

Comment:
unclear to me how much of this is being moved around – (all of it?) ­– versus new code; so, keep that in mind.

but, for one, any reason not to use the fixtures, _e.g._ `db_engine`?

Reply:
all of it's being moved around.  These mostly predate the fixtures, so they can be refactored but that wasn't the focus of this PR.

Reply:
Yep, I figured 👍